JASP-Engine/JASP/R/commonsummarystatsttestbayesian.R

```diff
@@ -361,7 +361,7 @@
     n2                        = 0 ,
     paired                    = FALSE,
     BF10user                  = BFlist[["BF10"]],
-    yAxisLegendRobustnessPlot = BFPlots,
+    yAxisLegendRobustnessPlot = bfPlotsH1H0,
```

Comment:
This causes the unit tests to fail. (The robustness plot expects a string. Can you make the necessary change?)